firebase-storage/build.gradle.kts

```diff
@@ -143,24 +143,6 @@ kotlin {
     }
 }
 
-if (project.property("firebase-storage.skipIosTests") == "true") {
```

Comment:
Is there anywhere these tests are still disabled when you run them locally. While the gradle script is fine for generating the CI matrix, we should still be able to run `./gradlew testIosArmSimulator64` without issue on a local checkout

Reply:
I propose two ways for running tests locally:

- create custom Gradle tasks (like `devIosTests`, `devJsTests`, `devJvmTests`) in root build.gradle.kts file. This tasks will use the EmulatorJobsMatrix.kt logic. As result many build.gradle.kts files will less code.

- EmulatorJobsMatrix.kt logic use the `gradle.properties` file for excluding jobs from matrix.

@Daeda88 What do you prefer for running tests locally?

Reply:
I think it should be the latter. Making custom gradle tasks to get back functionality that is there by default on gradle feels a bit weird to me.

You are right on having many build gradle scripts that do the same thing, but I think a better long term solution for that is to make a custom gradle plugin that does all the recurring setup. Not that I;m saying you should implement that mind you, just that if we want to reduce code, there is a better way.

Reply:
> I think it should be the latter. Making custom gradle tasks to get back functionality that is there by default on gradle feels a bit weird to me.

Agreed, the default test tasks should work as people expect

Reply:
updated